### DISCUSSION TITLE
Does not load on the website

### DISCUSSION THREAD

**Discussion by ALEXIScrazy78** [SkalskiP/make-sense#333] [Q&A] [answered]
![image](https://github.com/SkalskiP/make-sense/assets/139715153/351e621b-3b50-44df-a0d0-fa2730f559fc)

**Comment by SkalskiP** [comment, marked as answer]
Hi @ALEXIScrazy78 👋🏻 

We solved that issue already: https://github.com/SkalskiP/make-sense/issues/332 Let me know if you still experience any problems.
